src/components/portal/portal.js

```diff
@@ -1,35 +1,63 @@
 import React from 'react';
-import ReactPortal from 'react-portal';
 import PropTypes from 'prop-types';
+import { createPortal } from 'react-dom';
+import Browser from '../../utils/helpers/browser';
 
 class Portal extends React.Component {
-  static propTypes = {
-    /**
-     * Determine if the portal is visible or not
-     *
-     * @property open
-     * @type {Boolean}
-    */
-    open: PropTypes.bool,
 
+  static propTypes = {
     /**
      * The content of the portal.
      *
      * @property children
      * @type {Node}
      */
-    children: PropTypes.node
+    children: PropTypes.node,
+    /**
+     * The DomNode 
+     *
+     * @property node
+     * @type {Any}
+     */
+    node: PropTypes.any
+  }
+
+  componentWillUnmount() {
+    if (this.defaultNode) {
+      Browser.getDocument().body.removeChild(this.defaultNode);
+    }
+    this.defaultNode = null;
+  }
+
+  getDefaultDiv() {
+    
+    if (!this.props.node && !this.defaultNode && Browser.isDomAvailable()) {
+      this.defaultNode = Browser.getDocument().createElement('div');
+      this.defaultNode.classList.add('portal');
+      document.body.appendChild(this.defaultNode);
+    }
+    return this.defaultNode;
   }
 
   render() {
+    if (!Browser.isDomAvailable()) {
+      return null;
+    }
+
+    this.node = (this.props.node) ? this.props.node : this.getDefaultDiv();
```

Comment:
is there any benefit from storing this on `this.node`?

Reply:
I think this is my ignorance speaking. I seem to have unstood that we wanted to be able to access the Portal html node for testing purposes. There may be another way of access this through Enzyme

Reply:
looks like `wrapper.find` should work now?

Reply:
I suppose that I should take it out and we will see the need when it arises.